app/Http/Middleware/TrustProxies.php

```diff
@@ -12,7 +12,7 @@ class TrustProxies extends Middleware
      *
      * @var array|string
      */
-    protected $proxies;
+    protected $proxies = ['172.5.0.10'];
```

Comment:
What's this and why?

Reply:
We are running mars behind a reverse proxy and setting the reverse proxy as a trusted proxy for Laravel is mandatory if we want to use HTTPS. (https://laravel.com/docs/9.x/requests#configuring-trusted-proxies) There are some other hacky ways of setting it up but I didn't like them.
I would like it to be configurable from the .env, and I will add this to the PR.

Reply:
Okay, and add a comment here. Will this affect local development?

Reply:
I'm not 100% sure, and have not tested it, but I think it only comes in to effect if at least one of the headers in the request:
```
Request::HEADER_X_FORWARDED_FOR |
Request::HEADER_X_FORWARDED_HOST |
Request::HEADER_X_FORWARDED_PORT |
Request::HEADER_X_FORWARDED_PROTO |
Request::HEADER_X_FORWARDED_AWS_ELB
```
Which are mostly stuff that can come from a reverse proxy / load-balancer. But I think it should not have any effect unless you are connecting from IP `172.5.0.10` and the `APP_URL`'s protocol is https.

I will look into it today or tomorrow, but I think it should not have any effect for local development.

Reply:
I have tested it in my local environment and have not found any issues. So I think it should be fine

Reply:
Could we make this configurable instead of hard-coding the IP in the source?

Reply:
Yep, I would like to make it configurable, I just currently have a long lead-time

Reply:
Or if someone would fix it, I would happily give advice and / or check the code